about.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>about</title>
+    <script src="js/index.js"></script>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-T3c6CoIi6uLrA9TneNEoa7RxnatzjcDSCmG1MXxSR1GAsXEV/Dwwykc2MPK8M2HN" crossorigin="anonymous">
+    <link rel="stylesheet" href="./css/index.css" />
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@tabler/icons-webfont@latest/tabler-icons.min.css">
+    <link href='https://fonts.googleapis.com/css?family=Ubuntu' rel='stylesheet'>
+</head>
+
+<body>
+    <nav class="navbar navbar-controller">
+        <div class="container-fluid">
+            <a class="navbar-brand" href="#">
+                <div class="logo">R.</div>
+            </a>
+            <ul class="navbar nav">
+                <li class="nav-item">
+                    <div class="btn-group dropstart">
+                        <button type="button" class="btn btn-transparent" data-bs-toggle="dropdown">
+                            <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" class="bi bi-list"
+                                viewBox="0 0 16 16">
+                                <path fill-rule="evenodd"
+                                    d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z" />
+                            </svg>
+                        </button>
+                        <ul class="dropdown-menu">
+                            <li><a class="dropdown-item" href="index.html"><i class="ti ti-home-2"></i> Home</a></li>
+                            <li><a class="dropdown-item active" href="about.html"><i class="ti ti-info-circle"></i>Über Uns</a></li>
+                            <li><a class="dropdown-item" href="kontakt.html"><i class="ti ti-address-book"></i>Kontakt</a></li>
+                            <li><a class="dropdown-item" href="impressum.html"><i class="ti ti-section-sign"></i>Impressum</a></li>
```

Comment:
Here you used different languages. Let us decide to switch to English for now. Also for the files.